README.md

```diff
@@ -1,3 +1,10 @@
+The Atomic tool has been deprecated.
```

Comment:
```suggestion
# ⚠️ The Atomic tool has been deprecated. ⚠️
```

Reply:
Neat change, I didn't know you could do the warning signs like that.

Reply:
Do we have a date or a RHEL release tie in that the deprecation happened?